otwcode/otwarchive
# AO3-5834 Allow cache expiration on works and bookmarks to be configurable

# Pull Request Checklist


## Issue

https://otwarchive.atlassian.net/browse/AO3-5834
## Purpose

Sets the cache time on works and bookmarks to 30 minutes

## Testing Instructions

Lets set it to 30 minutes for both, using different variables.
Testing, look at the pages, make a change, see change not reflected wait 30 minutes and refresh the page see page changes.
